Xamarin.Forms.Platform.GTK/Animations/BaseAnimation.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Timers;
+
+namespace Xamarin.Forms.Platform.GTK.Animations
+{
+    public abstract class BaseAnimation
```

Comment:
Why do we want this public ?

Reply:
It is used in gtk widgets to perform certain animations. For example, with the MasterDetailPage. I do not think it's necessary to be public having all the animations API in Xamarin.Forms. Changed!.